test/sqllogictest/ldbc_bi.slt

```diff
@@ -0,0 +1,4022 @@
+            Project (#0, #2)
+              Filter (#1 > 2010-02-12 00:00:00 UTC)
+                Get materialize.public.top100popularforumsq04
+
+Used Indexes:
+  - materialize.public.person_id
+  - materialize.public.message_creatorpersonid
+  - materialize.public.top100popularforumsq04_id
+
+EOF
+
+######################################################################
+# QUERY 05
+######################################################################
+
+# \set tag '\'Sikh_Empire\''
+
+query T multiline
+EXPLAIN WITH detail AS (
+SELECT Message.CreatorPersonId AS CreatorPersonId
+     , sum(coalesce(Cs.c, 0))  AS replyCount
+     , sum(coalesce(Plm.c, 0)) AS likeCount
+     , count(Message.MessageId) AS messageCount
+  FROM Tag
+  JOIN Message_hasTag_Tag
+    ON Message_hasTag_Tag.TagId = Tag.id
+  JOIN Message
+    ON Message.MessageId = Message_hasTag_Tag.MessageId
+  LEFT JOIN (SELECT ParentMessageId, count(*) FROM Message c WHERE ParentMessageId IS NOT NULL GROUP BY ParentMessageId) Cs(id, c) ON Cs.id = Message.MessageId
+  LEFT JOIN (SELECT MessageId, count(*) FROM Person_likes_Message GROUP BY MessageId) Plm(id, c) ON Plm.id = Message.MessageId
+ WHERE Tag.name = 'Sikh_Empire'
+ GROUP BY Message.CreatorPersonId
+)
+SELECT CreatorPersonId AS "person.id"
+     , replyCount
+     , likeCount
+     , messageCount
+     , 1*messageCount + 2*replyCount + 10*likeCount AS score
+  FROM detail
+ ORDER BY score DESC, CreatorPersonId
+ LIMIT 100
+----
+Explained Query:
+  Finish order_by=[#4 desc nulls_first, #0 asc nulls_last] limit=100 output=[#0..=#4]
+    Return
+      Map (((bigint_to_numeric((1 * #3)) + (2 * #1)) + (10 * #2)))
+        Reduce group_by=[#0] aggregates=[sum(coalesce(#1, 0)), sum(coalesce(#2, 0)), count(*)]
+          Union
+            Map (null)
+              Union
+                Negate
+                  Project (#0, #1)
+                    Get l3
+                Project (#1, #2)
+                  Get l2
+            Get l3
+    With
+      cte l3 =
+        Project (#1, #2, #4)
+          Join on=(#0 = #3) type=differential
+            ArrangeBy keys=[[#0]]
+              Get l2
+            ArrangeBy keys=[[#0]]
+              Reduce group_by=[#0] aggregates=[count(*)]
+                Project (#2)
+                  Get materialize.public.person_likes_message
+      cte l2 =
+        Union
+          Map (null)
+            Union
+              Negate
+                Project (#0, #1)
+                  Get l1
+              Get l0
+          Get l1
+      cte l1 =
+        Project (#0, #1, #3)
+          Join on=(#0 = #2) type=differential
+            ArrangeBy keys=[[#0]]
+              Get l0
+            ArrangeBy keys=[[#0]]
+              Reduce group_by=[#0] aggregates=[count(*)]
+                Project (#12)
+                  Filter (#12) IS NOT NULL
+                    Get materialize.public.message
+      cte l0 =
+        Project (#4, #15)
+          Filter (#1 = "Sikh_Empire") AND (#0) IS NOT NULL
```

Comment:
If you created an index on `Tag.name`, this literal constraint would hopefully turn into a lookup in that index.